mobile.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM codait/max-base
+
+RUN mkdir -p /workspace/assets
+
+RUN pip install numpy && \
+    pip install tensorflow && \
+    pip install pillow
+
+RUN wget -nv --show-progress --progress=bar:force:noscroll http://max-assets.s3-api.us-geo.objectstorage.softlayer.net/deeplab/mobile.pb && \
```

Comment:
I think for consistency let's still use a `tar.gz` file even though it's only one file inside. I think @ptitzler had mentioned this approach for consistency